index.src.html

```diff
@@ -70,6 +70,10 @@
       text: Navigator
 spec: HTML; urlPrefix: https://html.spec.whatwg.org/multipage/
   type: dfn
+    text: the same; url: concept-origin-effective-domain
```

Comment:
I think this is only used once, and we should probably update it to something like "`If A is <a>same-origin</a> with B`".